uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/instances/leave-request.instance.js

```diff
@@ -11,6 +11,71 @@ define([
   instances.factory('LeaveRequestInstance', ['$q', 'checkPermissions', 'OptionGroup',
     'shared-settings', 'ModelInstance', 'LeaveRequestAPI',
     function ($q, checkPermissions, OptionGroup, sharedSettings, ModelInstance, LeaveRequestAPI) {
+      /**
+       * Amends balance change breakdown if calculation unit absence type is "hours".
+       * It changes the balance first and last days (only first day in case of single day request)
+       *   by setting the selected deductions.
+       * @NOTE this function mutates the "balanceChange" object
+       *
+       * @param  {Object} balanceChange
+       * @param  {String} calculationUnit
+       * @return {Promise} resolve with amended balance change in case of "hours" absence type
+       *   or returns unamended balance change in case of "days" absence type
+       */
+      function amendHourlyBalanceChangeBreakdown (balanceChange, calculationUnit) {
+        var breakdown = balanceChange.breakdown;
```

Comment:
is this var really necessary? isn't `balanceChange.breakdown` enough?

Reply:
It shortens code.

Reply:
Upd: I removed it.